astro/src/components/get-started/DockerInstallAndStart.astro

```diff
@@ -52,9 +52,9 @@ cd fusionauth-example-express-start-here`}/>
 
 <Code lang="shell" code="docker compose up -d" />
 
-You should see multiple containers start up; one for the database, one for FusionAuth and one for our mail server.
+<p>You should see multiple containers start up; one for the database, one for FusionAuth and one for our mail server.</p>
 
-<p>Here you are using a bootstrapping feature of FusionAuth called <a href="/docs/get-started/download-and-install/development/kickstart">Kickstart</a>. When FusionAuth comes up for the first time, it will look at the <code>kickstart/kickstart.json</code> file and configure FusionAuth to the specified state.</p>
+<p>Here you are using a bootstrapping feature of FusionAuth called <a href="/docs/get-started/download-and-install/development/kickstart">Kickstart</a>. When FusionAuth comes up for the first time, it will look at the <code>kickstart/kickstart.json</code> file and configure FusionAuth to the specified state.  In this case, it is adding an application and a couple of users which you'll use later in the guide.</p>
```

Comment:
```suggestion
<p>Here you are using a bootstrapping feature of FusionAuth called <a href="/docs/get-started/download-and-install/development/kickstart">Kickstart</a>. When FusionAuth comes up for the first time, it will look at the <code>kickstart/kickstart.json</code> file and configure FusionAuth to the specified state. In this case, it is adding an application and a couple of users which you'll use later in the guide.</p>
```